packages/addons-manager/metadata.yaml

```diff
@@ -3,6 +3,8 @@ kind: PackageMetadata
 metadata:
   name: addons-manager.tanzu.vmware.com
   namespace: addons-manager
+  annotations:
+    packaging.carvel.dev/revision: "1.0"
```

Comment:
Why do we start with revision 1.0 to begin with? And does this versioning applies to all packages in the repository?

Reply:
I think this is the first annotation we add for revision, and other pacakges without this annotation will be considered to have version -1 to avoid conflict

Reply:
correct. Since this is the first time we introduce this versionning, we use 1,.0 because it will allow any future changes to go higher or lower. We can't predict why the change would happen but we can allow for all options.